rdmd.d

```diff
@@ -785,7 +791,7 @@ addition to compiler options, rdmd recognizes the following options:
                      (multiple --extra-file allowed)
   --force            force a rebuild even if apparently not necessary
   --help             this message
-  --loop             assume \"foreach (line; stdin.byLine()) { ... }\" for eval
+  --loop=code        like eval, but wraps code in \"foreach (line; stdin.byLine()) { ... }\"
```

Comment:
Generally unrelated changes should at least be in separate commits.

Does this bring the help text in line with the man page, then? They should be in sync.